src/ResourceManager/MachineLearning/Commands.MachineLearning/Cmdlets/RemoveAzureMLWebService.cs

```diff
@@ -0,0 +1,92 @@
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.MachineLearning.Utilities;
+using Microsoft.Azure.Management.MachineLearning.WebServices.Models;
+using Microsoft.WindowsAzure.Commands.Common;
+
+namespace Microsoft.Azure.Commands.MachineLearning
+{
+    [Cmdlet(VerbsCommon.Remove, WebServicesCmdletBase.CommandletSuffix)]
+    [OutputType(typeof(void))]
+    public class RemoveAzureMLWebService : WebServicesCmdletBase
+    {
+        protected const string RemoveByNameGroupParameterSet = 
+            "Remove an Azure ML web service resouce by name and resource group.";
+        protected const string RemoveByObjectParameterSet = 
+            "Remove an Azure ML web service specified as an object.";
+
+        [Parameter(
+            ParameterSetName = RemoveAzureMLWebService.RemoveByNameGroupParameterSet, 
+            Mandatory = true, 
+            HelpMessage = "The name of the resource group for the Azure ML web service.")]
+        [ValidateNotNullOrEmpty]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(
+            ParameterSetName = RemoveAzureMLWebService.RemoveByNameGroupParameterSet, 
+            Mandatory = true, 
+            HelpMessage = "The name of the web service.")]
+        [ValidateNotNullOrEmpty]
+        public string Name { get; set; }
+
+        [Parameter(
+            ParameterSetName = RemoveAzureMLWebService.RemoveByObjectParameterSet, 
+            Mandatory = true, 
+            HelpMessage = "The machine learning web service object.", 
+            ValueFromPipeline = true)]
+        [ValidateNotNullOrEmpty]
+        public WebService MlWebService { get; set; }
+
+        [Parameter(Mandatory = false, HelpMessage = "Do not ask for confirmation.")]
+        public SwitchParameter Force { get; set; }
+
+        protected override void RunCmdlet()
+        {
+            if (string.Equals(
+                        this.ParameterSetName, 
+                        RemoveAzureMLWebService.RemoveByObjectParameterSet, 
+                        StringComparison.OrdinalIgnoreCase))
+            {
+                string subscriptionId, resourceGroup, webServiceName;
+                if (!CmdletHelpers.TryParseMlWebServiceMetadataFromResourceId(
+                                    this.MlWebService.Id, 
+                                    out subscriptionId, 
+                                    out resourceGroup, 
+                                    out webServiceName))
+                {
+                    throw new ValidationMetadataException(Resources.InvalidWebServiceIdOnObject);
+                }
+
+                this.ResourceGroupName = resourceGroup;
+                this.Name = webServiceName;
+            }
+
+            this.ConfirmAction(
```

Comment:
It is not. ConfirmAction implements this incorrectly and is being changed. Please implement as I have specified.  There is no need for -Force if it is *always* applied.